src/types/register_request.rs

```diff
@@ -24,6 +23,49 @@ pub enum RegisterRequest {
 pub struct RegisterInputRequest {
     pub input_id: InputId,
     pub port: Port,
+    pub video: Option<Video>,
+    pub audio: Option<Audio>,
+}
+
+#[derive(Debug, Serialize, Deserialize, Clone, JsonSchema)]
+pub struct Video {
+    #[serde(default)]
+    pub codec: VideoCodec,
+    /// Default 96
+    pub rtp_payload_type: Option<u8>,
+}
+
+#[derive(Debug, Default, Serialize, Deserialize, Clone, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub enum VideoCodec {
+    #[default]
+    H264,
+}
+
+#[derive(Debug, Serialize, Deserialize, Clone, JsonSchema)]
```

Comment:
We do not generate documentation for input streams now, but treat it as if we would. If we plan to add new stream types they will have to be generated